.github/workflows/ci-changelog.yml

```
@@ -0,0 +1,67 @@
+jobs:
+  changelog-sync:
+    name: changelog sync
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: sync assets to S3
+        continue-on-error: true
+        if: ${{ env.AWS_SECRET_ACCESS_KEY != '' && matrix.builds.runs-on != 'self-hosted' }}
```

actionlint (1.7.4)

property "builds" is not defined in object type {}

(expression)